docs/VnVPlan/VnVPlan.tex

```diff
@@ -724,16 +728,185 @@ \subsubsection{System Behaviour and Recommendations}
+					
+  Input: The user tries changing the application's font size.
+					
+  Output: The font size should be changed throughout the application without negatively impacting readability
+  or general functionality.
+  
+  How test will be performed: Testers will be asked to change the font size to their desired size and
+  navigate through the application to check usability.
+
+  \item{NFRT-UH4\\}
+
+  Type: Manual, Dynamic
+					
+  Initial State: The application is opened on a device and logged in to a user's account with internet access available.
+					
+  Input: The user changes the accessibility colour scheme.
+					
+  Output: The colour scheme should be changed throughout the application without negatively impacting readability
+  or general functionality.
+  
+  How test will be performed: Testers will be asked to change the accessibility colour scheme to their desired setting and
+  navigate through the application to check usability.
+
+  \item{NFRT-UH5\\}
+
+  Type: Manual, Dynamic
+					
+  Initial State: The application is opened on a device and logged in to a user's account with accessibility settings
+  having been changed.
+					
+  Input: The user closes the application.
+					
+  Output: When re-opened, the application should still be logged into the user's account and accessibility
+  settings should still be in place.
+  
+  How test will be performed: Following \textit{NFRT-UH4} and \textit{NFRT-UH5}, testers will be asked to
+  close the application on their device while logged in and re-open it afterwards.
+
+  \item{NFRT-UH6\\}
+
+  Type: Manual
 
-\wss{Tests related to usability could include conducting a usability test and
-  survey.  The survey will be in the Appendix.}
+  How test will be performed: A survey will be presented to testers to evaluate the overall usability of the
+  application after first going through \textit{NFRT-UH1}. Afterwards, the team will show them any remaining features
+  they had not yet experienced and given the opportunity to freely experiment with the applications. A survey will then be
+  provided to them. The questions asked will be as written in Section 6.4 of this document.
+\end{enumerate}
+
+\subsubsection{Performance Requirements}
+
+\begin{enumerate}
+  \item{NFRT-PR1\\}
+
+  Type: Manual, Dynamic
+
+  How test will be performed: All pages of the application will be navigated through while using the
+  \textit{Flutter Performance Profiling} tools. Times for rendering and frontend loading will then be recorded
+  to ensure that frontend performance is within expectation.
+
+  \item{NFRT-PR2\\}
+
+  Type: Automatic, Dynamic, Manual
+
+  How test will be performed: Requests will be sent to all backend endpoints via \textit{Postman} and after
+  all requests are sent, the times for each request will be recorded. The team will then manually confirm that
+  the backend performs within expectation.
```

Comment:
is this really Manual? seems mostly automatic to me ... the part about the team manually looking at the times and checking if it is within expectation can be automated easily and even if its not, its still not enough to warrent being labeled as manual.